x-pack/plugin/security/qa/security-basic/src/javaRestTest/java/org/elasticsearch/xpack/security/SecuritySettingsIT.java

```diff
@@ -72,6 +72,27 @@ public void testBasicWorkflow() throws IOException {
         assertThat(mapView.get("security.index.auto_expand_replicas"), equalTo("0-all"));
     }
 
+    public void testTierPreference() throws IOException {
+        Request req = new Request("PUT", "/_security/settings");
+        req.setJsonEntity("""
+            {
+                "security": {
+                    "index.routing.allocation.include._tier_preference": "data_hot"
+                },
+                "security-profile": {
+                    "index.routing.allocation.include._tier_preference": "data_hot"
+                }
+            }
+            """);
+        Response resp = adminClient().performRequest(req);
+        assertOK(resp);
+        Request getRequest = new Request("GET", "/_security/settings");
```

Comment:
should this return the default values for all 3 not explicitly set ?

Reply:
No, it's basically a thin proxy over the regular settings API.